components/Settings.js

```diff
@@ -95,6 +96,11 @@ function WindowSettings({
           />
         </div>
       )}
+      <Toggle
+        label="Whitespace"
```

Comment:
You can leave the variable as whitespace, but can you change the `label` to `"Hidden characters"`?